src/lib/style/components/_bar-loader.scss

```
@@ -5,10 +5,18 @@
   height: 30px;
   width: 100%;
   .bar-background-color {
-    background-color: rgba(49, 95, 124, 0.05);
+    // background-color: rgba(49, 95, 124, 0.05);
+    background-color: #DFE2E6;
```

```suggestion
    background-color:  map.get(SRC.$colors, 'gray-400');
```

You will need to import the `variables.scss` file as `SWC` in this case. We do it all over the place in SCSS but let me know if you have trouble finding it.